plugins/cinema4d/import_cast.pyp

```diff
@@ -173,6 +177,7 @@ def importModelNode(doc, node, model, path):
         for i in range(mesh.UVLayerCount()):
             uvTag = c4d.UVWTag(facesCount)
             uvBuffer = mesh.VertexUVLayerBuffer(0)
```

Comment:
I just noticed you're not importing the correct layer, you're using layer 0 for every UV layer atm!

`mesh.VertexUVLayerBuffer(i)`

Reply:
Oh yeah, that's something I forgot to change for testing. 
Cinema 4Ds materials always use the latest added UVW tag, so some models I tested had wrong UVs then and I couldn't work on materials. 
Thanks for letting me know!